src/plugins/intel_gpu/tests/test_cases/non_zero_gpu_test.cpp

```diff
@@ -15,6 +15,19 @@
 using namespace cldnn;
 using namespace ::tests;
 
+template<typename T>
+std::vector<T> generate_random_input(const size_t input_size, int min, int max) {
+    static std::default_random_engine generator(random_seed);
+    int k = 8; // 1/k is the resolution of the floating point numbers
+    std::uniform_int_distribution<int> distribution(k * min, k * max);
+    std::vector<T> v(input_size);
+    for (size_t i = 0; i < input_size; ++i) {
+        v[i] = (T)distribution(generator);
+        v[i] /= k;
+    }
+    return v;
+}
```

Comment:
Can we ensure that the generated values have "0"s definitely? 
How about 
1) generate with original utility generate_random
2) select random number of "# of zeros" // count_zero (e.g., 10~100 zeros)
3) generate random "indices"  to set input mem to be zero
4) set the value of those indices to zero

Reply:
Update random generation input function
- Use original random generation util function
- Set minimum number of zero inputs
- Generate random indices to set zero in input buffer